### PR TITLE
Fee table update

### PR DESCRIPTION
**Pull request by mstrug-rdx**
## Summary
Updated fee table basing on results of workflow run: https://github.com/radixdlt/radixdlt-scrypto/actions/runs/6236179628

### CODE REVIEW

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 1]
```diff
@@ -172,23 +172,22 @@ impl FeeTable {
     #[inline]
     pub fn before_invoke_cost(&self, _actor: &Actor, input_size: usize) -> u32 {
         add(
-            13 / CPU_INSTRUCTIONS_TO_COST_UNIT,
+            10 / CPU_INSTRUCTIONS_TO_COST_UNIT,
```
Comment (mstrug-rdx) [inline radix-engine/src/system/system_modules/costing/fee_table.rs:175]: This equals 0, should we just change this to 1?

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 2]
```diff
@@ -172,23 +172,22 @@ impl FeeTable {
...
             Self::data_processing_cost(input_size),
         )
     }
 
     #[inline]
     pub fn after_invoke_cost(&self, input_size: usize) -> u32 {
-        // used max cpu instruction counts
         add(
-            4756 / CPU_INSTRUCTIONS_TO_COST_UNIT,
+            6 / CPU_INSTRUCTIONS_TO_COST_UNIT,
```
Comment (mstrug-rdx) [inline radix-engine/src/system/system_modules/costing/fee_table.rs:183]: This equals 0, should we just change this to 1?

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 3]
```diff
@@ -316,13 +316,13 @@ impl FeeTable {
         _partition_number: &PartitionNumber,
         _substate_key: &SubstateKey,
     ) -> u32 {
-        1904
```
Comment (mstrug-rdx) [inline radix-engine/src/system/system_modules/costing/fee_table.rs:319]: This is not converted to cost units.

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 4]
```diff
@@ -211,7 +210,7 @@ impl FeeTable {
 
     #[inline]
     pub fn pin_node_cost(&self, _node_id: &NodeId) -> u32 {
-        2473
```
Comment (mstrug-rdx) [inline radix-engine/src/system/system_modules/costing/fee_table.rs:214]: This is not converted to cost units.

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 5]
```diff
@@ -286,15 +286,15 @@ impl FeeTable {
...
+            CloseSubstateEvent::Start(..) => 4390 / CPU_INSTRUCTIONS_TO_COST_UNIT,
         }
     }
 
     #[inline]
     pub fn set_substate_cost(&self, event: &SetSubstateEvent) -> u32 {
         match event {
             SetSubstateEvent::Start(.., value) => add(
-                9750 / CPU_INSTRUCTIONS_TO_COST_UNIT,
+                4530 / CPU_INSTRUCTIONS_TO_COST_UNIT,
```
Comment (mstrug-rdx) [inline radix-engine/src/system/system_modules/costing/fee_table.rs:297]: This is the highest decrease of value.

File: radix-engine/src/system/system_modules/costing/fee_table.rs (hunk 3 above)
Comment (iamyulong) [reply radix-engine/src/system/system_modules/costing/fee_table.rs:319]: Nice spot!

File: assets/native_function_base_costs.csv [hunk 6]
```diff
@@ -1,225 +1,225 @@
...
-0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,set_default_deposit_rule,2202486
-0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,set_resource_preference,2421801
-0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,try_deposit_batch_or_abort,4378415
-0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,try_deposit_batch_or_refund,3517108
-0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,try_deposit_or_abort,3523273
-0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,try_deposit_or_refund,3239149
-0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,withdraw,2065880
-0d906318c6318c6ee313598c6318c6318cf7bcaa2e954a9626318c6318c6,withdraw_non_fungibles,2863618
+0d906318c6318c60f716464c6318c6318cf7bfcad6a3152b46318c6318c6,PackageRoyalty_claim_royalties,3839799
+0d906318c6318c60f716464c6318c6318cf7bfcad6a3152b46318c6318c6,publish_wasm,350095960
```
Comment (iamyulong) [inline assets/native_function_base_costs.csv:2]: Quite an increase for `publish_wasm`?

File: radix-engine/src/system/system_modules/costing/fee_table.rs (hunk 1 above)
Comment (iamyulong) [reply radix-engine/src/system/system_modules/costing/fee_table.rs:175]: Which function is the `10` derived from?

File: radix-engine/src/system/system_modules/costing/fee_table.rs (hunk 2 above)
Comment (iamyulong) [reply radix-engine/src/system/system_modules/costing/fee_table.rs:183]: Which is the `6` derived from?

File: radix-engine/src/system/system_modules/costing/fee_table.rs (hunk 4 above)
Comment (iamyulong) [reply radix-engine/src/system/system_modules/costing/fee_table.rs:214]: Nice catch!

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 7]
```diff
@@ -225,18 +224,19 @@ impl FeeTable {
...
         }
     }
 
     #[inline]
+    // TODO: Rename to 'create_node_from_cost' as this event is emited from kernel_create_node_from()
     pub fn move_module_cost(&self, event: &MoveModuleEvent) -> u32 {
         match event {
             MoveModuleEvent::IOAccess(io_access) => add(
-                4791 / CPU_INSTRUCTIONS_TO_COST_UNIT,
+                138153 / CPU_INSTRUCTIONS_TO_COST_UNIT,
```
[the remark this answers is not in the document]
Comment (iamyulong) [reply radix-engine/src/system/system_modules/costing/fee_table.rs:239]: Again, which function is this number derived from? `create_node_from_cost` issues multiple events.

**Comment by iamyulong** [comment]
Discussed. 

We're going to merge the PR after the following changes:
- Revert the change to `move_module_cost`
- Re-calculate the costing for invocation based on the data we've collected

**Review by iamyulong — APPROVED** [review]

**Pull request merged**